### PR TITLE
fix(core): Do not block app startup by telemetry (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
This was added in https://github.com/n8n-io/n8n/pull/5080/files#diff-e1c41b1f8cc627242377b30fbee7af28472c81abe5f4f41da1c513f264f128e2L1754.
Reverting this part of the code to how it was before #5080

### CODE REVIEW

File: packages/cli/src/Server.ts [hunk 1]
```diff
@@ -1494,10 +1494,11 @@ export async function start(): Promise<void> {
...
 	initEvents();
 
-	const workflow = await Db.collections.Workflow.findOne({
+	void Db.collections.Workflow.findOne({
 		select: ['createdAt'],
 		order: { createdAt: 'ASC' },
 		where: {},
-	});
-	await InternalHooksManager.getInstance().onServerStarted(diagnosticInfo, workflow?.createdAt);
+	}).then(async (workflow) =>
```
Comment (ivov) [inline packages/cli/src/Server.ts:1501]:
```suggestion
	}).then((workflow) =>
```

**Review by ivov — APPROVED** [review]
Much better, thank you!

File: packages/cli/src/Server.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/Server.ts:1501]: eslint complains about the missing `async` then.

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**